hail/python/hail/backend/service_backend.py

```diff
@@ -425,14 +429,12 @@ async def _rpc(self,
                         await write_str(infile, bucket)
                         await write_str(infile, mount_point)
                         await write_bool(infile, readonly)
-                    await inputs(infile, token)
+                    await inputs(infile, self._batch.token)
```

Comment:
This has been incorrect since we kept the Batch longer than the `_rpc` method's lifetime. The token would get regenerated repeatedly even though the batch never changed.